package.xml

```diff
@@ -14,6 +14,7 @@
 
   <buildtool_depend>catkin</buildtool_depend>
 
+  <exec_depend>depthimage_to_laserscan</exec_depend>
```

Comment:
This is a launch dependency, not a software dependency, so should not be in the `package.xml`